File: src/data/snippets/strings/snippet.java
public static void main(String[] args) {
    // String creation
    String str1 = "Hello";
    String str2 = "World";

    // String concatenation
    String combined = str1 + " " + str2;
    System.out.println("Combined String: " + combined);

    // String length
    System.out.println("Length: " + combined.length());

    // String indexing
    char firstChar = str1.charAt(0);
    System.out.println("First character of str1: " + firstChar);

    // String slicing
    String substring = str2.substring(1, 4); // From index 1 to 3
    System.out.println("Substring of str2: " + substring);

    // Check for substring
    boolean contains = str1.contains("lo");
    System.out.println("Does str1 contain 'lo'? " + (contains ? "Yes" : "No"));
}

File: src/data/snippets/strings/snippet.cs

// String creation
string str1 = "Hello";
string str2 = "World";

// String concatenation
string combined = str1 + " " + str2;
Console.WriteLine("Combined String: " + combined);

// String length
Console.WriteLine("Length: " + combined.Length);

// String indexing
char firstChar = str1[0];
Console.WriteLine("First character of str1: " + firstChar);

// String slicing
string substring = str2.Substring(1, 3); // From index 1, length 3
Console.WriteLine("Substring of str2: " + substring);

// Check for substring
bool contains = str1.Contains("lo");
Console.WriteLine("Does str1 contain 'lo'? " + (contains ? "Yes" : "No"));
